src/js/routes/factories/network.js

```diff
@@ -111,7 +111,8 @@ let RouteFactory = {
                 parentCrumb: 'virtual-networks-tab-detail-tasks-detail',
                 getCrumbs: function () { return []; }
               }
-            }
+            },
+            title: 'Details'
```

Comment:
Yeah, maybe we should derive the title from a constant object that maps the route name to the tab's title. Just a thought.